src/gui/optionsdialog.ui

```diff
@@ -1174,8 +1202,11 @@
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_2">
+                   <property name="toolTip">
+                    <string>Receiver</string>
+                   </property>
                    <property name="text">
-                    <string comment="To receiver">To:</string>
```

Comment:
I just checked in Qt Linguist, using `comment` field for each of the "To:" strings results in 2 entries for translation, this is desired as different language could have used different words under different conditions. If we switched to `extracomment`, Qt Linguist would only provide 1 entry for translation.
In short, I think we need to revert `extracomment` back to `comment` but only for "To:" and "From:" strings.
https://github.com/qbittorrent/qBittorrent/issues/11064

Reply:
Done.